README.md

```diff
@@ -106,6 +107,19 @@ TypeDoc is used for documentation. You can generate HTML locally with the follow
 
 ```npm run doc```
 
+## Getting Help
+
+Please use these community resources for getting help.
+* Ask a question on StackOverflow and tag it with the [amazon-qldb](https://stackoverflow.com/questions/tagged/amazon-qldb) tag.
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
```

Comment:
Missing the entry for the QLDB Forum that's in the Python version.

Reply:
Added